students/arkadiusz_kasprzyk/lesson_07_strings/swap_words.py

```diff
@@ -0,0 +1,11 @@
+# swap_words.py
+
+
+def swap_words(string):
+    k = string.find(" ")
+    return string[(k + 1):] + " " + string[:k]
```

Comment:
A clearer way would be:
```
first, second = string.split(' ')
return ' '.join([second, first])
```

Reply:
I cannot disagree :) But it's opportunity to ask which is faster?

Reply:
`timeit`

Reply:
Using `.split()` and `.join()` gives
`486 ns ± 1.6 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)`
while the previous solution:
`522 ns ± 7.32 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)`
Case solved: use methods - they are clearer and faster.
Finally I have made all corrections and now... I'm waiting for the commit :)